core/deployment/src/main/java/io/quarkus/deployment/builditem/FeatureBuildItem.java

```diff
@@ -13,6 +13,7 @@
     public static final String AMAZON_LAMBDA = "amazon-lambda";
     public static final String ARTEMIS_CORE = "artemis-core";
     public static final String ARTEMIS_JMS = "artemis-jms";
+    public static final String AWS_XRAY = "aws-xray";
```

Comment:
So now I have a bit of a dilemma because we should probably have a consistent prefix between lambda and xray :).

I don't think we will change the extension name, it's a bit late for that probably but maybe we could change the feature names.

AFAIU, the correct naming scheme is AWS Lambda and AWS XRay.

@patriot1burke @evanchooly